Fix build on x86_64 with -Werror (implicit conversion from long to int)

--- src/version.cc~	2010-10-14 13:57:11.000000000 +0200
+++ src/version.cc	2011-10-08 16:56:14.000000000 +0200
@@ -38,13 +38,13 @@
       boost::split (headerVersion, header_version, boost::is_any_of("."));
       boost::split (libraryVersion, version, boost::is_any_of("."));
 
-      unsigned s = std::max (headerVersion.size (), libraryVersion.size ());
+      long unsigned s = std::max (headerVersion.size (), libraryVersion.size ());
       headerVersion.resize (s);
       libraryVersion.resize (s);
 
       assert (headerVersion.size () == libraryVersion.size ());
 
-      for (unsigned i = 0; i < s; ++i)
+      for (long unsigned i = 0; i < s; ++i)
 	{
 	  if (headerVersion != libraryVersion)
 	    return (headerVersion > libraryVersion) ? -1 : 1;
